pkg/distributor/distributor_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/efficientgo/core/errors"
```

Comment:
We use `"github.com/pkg/errors"` package instead.

Reply:
This was imported by mistake. Thank you.